demo/stripes.js

```diff
@@ -1,47 +1,27 @@
 #!/usr/bin/env node
 
 const commander = require('commander');
-const webpack = require('webpack');
-const HardSourceWebpackPlugin = require('hard-source-webpack-plugin');
 const path = require('path');
-const fs = require('fs');
-const nodeObjectHash = require('node-object-hash');
-const express = require('express');
-const webpackDevMiddleware = require('webpack-dev-middleware');
-const webpackHotMiddleware = require('webpack-hot-middleware');
-const connectHistoryApiFallback = require('connect-history-api-fallback');
-
-const StripesConfigPlugin = require('@folio/stripes-core/webpack/stripes-config-plugin');
-
-const stripesCorePath = path.dirname(require.resolve('@folio/stripes-core/index.html'));
-const cwd = path.resolve();
-const cwdModules = path.join(cwd, 'node_modules');
-const coreModules = path.join(stripesCorePath, 'node_modules');
-
+const stripes = require('@folio/stripes-core/webpack/stripes-node-api');
 const packageJSON = require('@folio/stripes-core/package.json');
+const webpack = require('webpack');
+const fs = require('fs');
 
 commander.version(packageJSON.version);
 
-const cachePlugin = new HardSourceWebpackPlugin({
-  cacheDirectory: path.join(cwd, 'webpackcache'),
-  recordsPath: path.join(cwd, 'webpackcache/records.json'),
-  configHash(webpackConfig) {
-    // Build a string value used by HardSource to determine which cache to
-    // use if [confighash] is in cacheDirectory or if the cache should be
-    // replaced if [confighash] does not appear in cacheDirectory.
-    return nodeObjectHash().hash(webpackConfig);
-  }
-});
-
 // Display webpack output to the console
 function processStats(err, stats) {
   if (err) {
     console.error(err); // eslint-disable-line no-console
   }
   console.log(stats.toString({ // eslint-disable-line no-console
     chunks: false,
-    colors: true
+    colors: true,
```

Comment:
this isn't an eslint issue?  thought we had no trailing commas enforced

Reply:
Nope, we just have `comma-dangle` turned off completely, so our repo has no opinion on the matter. I copied and pasted this from `stripes-core`, which does require dangling commas and enforces it.

Reply:
🎵  Mr. Code Dangles.... 🎵

Reply:
🆒  thanks for the explanation